test/typecheck/main.go

```diff
@@ -310,7 +328,9 @@ func main() {
 		args = append(args, ".")
 	}
 
-	c := collector{}
+	c := collector{
+		ignoreDirs: append(strings.Split(*ignoreDirs, ","), standardIgnoreDirs...),
```

Comment:
same question about appending an empty string of no additional ignore dirs are specified

Reply:
`strings.Split` returns an empty slice in this case

Reply:
https://github.com/kubernetes/kubernetes/pull/85457#discussion_r354126681 🤦‍♂